packages/components/src/components/radio-group/radio-group.jsx

```diff
@@ -11,8 +11,22 @@ const ItemWrapper = ({ children, should_wrap_items }) => {
     return children;
 };
 
-const RadioGroup = ({ className, name, onToggle, required, selected, should_wrap_items, children }) => {
-    const [selected_option, setSelectedOption] = React.useState(selected);
+const RadioGroup = ({
+    is_left,
+    is_center,
+    className,
+    name,
+    onToggle,
+    required,
+    selected,
+    should_wrap_items,
+    children,
+}) => {
+    const [selected_option, setSelectedOption] = React.useState(null);
```

Comment:
could you explain these changes? I mean default value for useState and implementing here useEffect

Reply:
the prop value will be passed to the component and set when the user re-visits the section. To do this useEffect is used. Also when the user makes changes to the section, that value need to be captured hence useState is used